lms/djangoapps/courseware/views.py

```diff
@@ -423,6 +437,11 @@ def _index_bulk_op(request, course_key, chapter, section, position):
 
         context['show_chat'] = show_chat
 
+        if settings.FEATURES.get('ENTRANCE_EXAMS', False) and getattr(course, 'entrance_exam_enabled', False):
```

Comment:
nitpick: entrance_exam_enabled is always going to be an attribute on a CourseDescriptor since it is an xBlockField, with a default of False. getattr() is not necessary, we could just do 'course.entrance_exam_enabled'